src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/SaveClaimWithCoreCaseDataStoreTest.java

```diff
@@ -0,0 +1,122 @@
+package uk.gov.hmcts.cmc.claimstore.controllers;
+
+import feign.FeignException;
+import org.junit.Test;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.web.servlet.MvcResult;
+import uk.gov.hmcts.cmc.claimstore.BaseSaveTest;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.ClaimData;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.cmc.claimstore.utils.ResourceLoader.successfulCoreCaseDataStoreStartResponse;
+import static uk.gov.hmcts.cmc.claimstore.utils.ResourceLoader.successfulCoreCaseDataStoreSubmitResponse;
+
+@TestPropertySource(
+    properties = {
+        "feature_toggles.core_case_data=true"
+    }
+)
+public class SaveClaimWithCoreCaseDataStoreTest extends BaseSaveTest {
+
+    private static final String SERVICE_TOKEN = "S2S token";
+
+    @Test
+    public void shouldStoreSealedCopyOfNonRepresentedClaimIntoCCDStore() throws Exception {
```

Comment:
Maybe `shouldStoreNonRepresentedClaimIntoCCDStore`? We use sealed copy term to refer to PDF documents.

Reply:
done